docs/python-reference/units.ipynb

```diff
@@ -64,11 +80,141 @@
     "sc.Unit('m/s')"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.Unit('count')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To convert data between compatible units, such as `m` to `mm`, see [sc.to_unit](../generated/scipp.to_unit.html#scipp.to-unit)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.to_unit(1 * sc.Unit('m'), 'mm')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Supported Units\n",
+    "\n",
+    "### Base Units\n",
+    "All SI base units are supported with the following names:\n",
+    "\n",
+    "Name | Unit\n",
+    "---|---\n",
+    "'m' | meter\n",
+    "'s' | second\n",
+    "'kg' | kilogram\n",
+    "'K' | kelvin\n",
+    "'A' | ampere\n",
+    "'mol' | mole\n",
+    "'cd' | candela"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sc.Unit('K')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition, these base units are supported for cases not covered by SI:\n",
+    "\n",
+    "name | Unit\n",
+    "---|---\n",
+    "'rad' | radian\n",
+    "'count' | single object counting\n",
+    "\n",
+    "### Derived units\n",
+    "A great number of derived units can also be specified as arguments to `sc.Unit`.\n",
+    "Some examples are\n",
+    "\n",
+    "Name | Unit\n",
+    "---|---\n",
+    "'Hz' | hertz\n",
+    "'J' | joule\n",
+    "'V' | volt\n",
+    "'W' | watt\n",
+    "'angstrom' / 'Å' | ångström\n",
+    "'L' | liter\n",
+    "'min' | minute\n",
+    "'D' / 'day' | day\n",
+    "\n",
+    "Units can be modified with SI prefixes, for instance"
```

Comment:
Might be worth mentioning also `(m)eV`, since this is very common in our community.